internal/cmd/base/option.go

```diff
@@ -83,3 +88,17 @@ func WithSkipTargetCreation() Option {
 		o.withSkipTargetCreation = true
 	}
 }
+
+// WithContainerImage tells the command which container image
+// to start a dev database with
+func WithContainerImage(name string) Option {
+	return func(o *Options) {
+		o.withContainerImage = name
+	}
+}
+
+func withDialect(dialect string) Option {
```

Comment:
Is there a reason you do not want to export this option?  If you do export please add godoc, otherwise this is fine